filescppgen.py

```diff
@@ -44,9 +41,10 @@ def usage():
                                            org="ome")
 repl["@COMMON_TAG_URL@"] = get_tag_url("ome-common-cpp",
                                        common_version, org="ome")
-repl["@XML_TAG_URL@"] = get_tag_url("bioformats", bf_version)
+repl["@MODEL_TAG_URL@"] = get_tag_url("ome-model", model_version)
```

Comment:
See https://ci.openmicroscopy.org/view/Release/job/OME-FILES-CPP-DEV-release-downloads/128/console, this should be `org="ome"`